romancal/stpipe/core.py

```diff
@@ -8,6 +8,8 @@ class RomanStep(Step):
     Base class for Roman calibration pipeline steps.
     """
 
+    output_ext =  '.asdf'
```

Comment:
I think we should add a spec string here:
```python
output_ext         = string(default='.asdf')     # Default type of output
```
This will automatically override the `stpipe.Step` definition of the parameter and allow us to merge this PR without waiting for an stpipe release.

Reply:
This means adding configobj. Is this where we are going with romancal?

Reply:
Eventually we'll want to convert this to a traitlet, but that's a ways off.  The class attribute gets overwritten by `stpipe.Step` when the step is instantiated:

```python
In [1]: from romancal.stpipe import RomanStep

In [2]: s = RomanStep()
2021-02-16 10:21:39,225 - stpipe.RomanStep - INFO - RomanStep instance created.

In [3]: s.output_ext is None
Out[3]: True
```